common/metrics/otel_metrics_handler.go

```diff
@@ -68,7 +68,7 @@ func (omp *otelMetricsHandler) WithTags(tags ...Tag) Handler {
 func (omp *otelMetricsHandler) Counter(counter string) CounterIface {
 	c, err := omp.provider.GetMeter().Int64Counter(counter)
 	if err != nil {
-		omp.l.Fatal("error getting metric", tag.NewStringTag("MetricName", counter), tag.Error(err))
+		omp.l.Error("error getting metric", tag.NewStringTag("MetricName", counter), tag.Error(err))
```

Comment:
Maybe we should return here and other cases?

Reply:
Oh, whoops. Yeah, we definitely should.